projects/packages/waf/src/brute-force-protection/class-brute-force-protection-blocked-login-page.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Class used to define Brute Force Protection Blocked Login Page.
+ *
+ * @package automattic/jetpack-waf
+ */
+
+namespace Automattic\Jetpack\Waf\Brute_Force_Protection;
+
+use Automattic\Jetpack\Redirect;
+use Automattic\Jetpack\Waf\Blocked_Login_Page;
+
+/**
+ * Brute Force Protection Blocked Login Page class.
+ */
+class Brute_Force_Protection_Blocked_Login_Page extends Blocked_Login_Page {
```

Comment:
Introducing these improvements while keeping the `Automattic\Jetpack\Waf\Brute_Force_Protection\Brute_Force_Protection_Blocked_Login_Page` class stable is great. No backwards compat issues or deprecations needed. :+1: